source/data-api/generated-endpoints.txt

```diff
@@ -173,23 +173,12 @@ Data API Requests <data-api-authenticate-requests>`.
    .. tab:: Application
       :tabid: auth-credentials
 
-      Application authentication requires users to authenticate each
-      Data API request by providing login credentials for one of the
-      following supported authentication providers:
-      
-      - :ref:`Email/Password <email-password-authentication>`
-      - :ref:`API Key <api-key-authentication>`
-      - :ref:`Custom JWT <custom-jwt-authentication>`
-      
-      To see how to include authentication credentials in Data API
-      requests, see :ref:`data-api-authenticate`.
-      
-      .. note::
-         
-         You must :ref:`enable an authentication provider
-         <authentication-providers>` before users can authenticate with
-         it.
-   
+      Application authentication requires users to log in with an
```

Comment:
same comments as for the equivalent paragraph on the custom endpoints page:

- https://github.com/mongodb/docs-app-services/pull/336/files#r1095060409
- https://github.com/mongodb/docs-app-services/pull/336/files#r1095063736